www/js/check_terms_version.js

```diff
@@ -0,0 +1,26 @@
+function checkTermsVersion (termsVersion) {
+
+  var templateHTML = app.templates.checkTermsVersionTemplate({});
```

Comment:
Think this name makes more sense.
```suggestion
  var templateHTML = app.templates.termsVersionTemplate({});
```